File: app/src/main/java/com/kota/Bahamut/Command/BahamutCommandLoadMoreArticle.java
package com.kota.Bahamut.Command;

import com.kota.Telnet.Reference.TelnetKeyboard;
import com.kota.Telnet.TelnetClient;

public class BahamutCommandLoadMoreArticle {
    public void execute() {
        System.out.print(toString() + " ");
        TelnetClient.getClient().sendKeyboardInputToServerInBackground(TelnetKeyboard.PAGE_DOWN, 1);
    }

    public String toString() {
        return "[LoadMoreArticle]";
    }
}

File: app/src/main/java/com/kota/Bahamut/command/BahamutCommandLoadMoreArticle.kt
package com.kota.Bahamut.command

import com.kota.telnet.reference.TelnetKeyboard
import com.kota.telnet.TelnetClient

class BahamutCommandLoadMoreArticle {
    fun execute() {
        print(toString() + " ")
        TelnetClient.client?.sendKeyboardInputToServerInBackground(TelnetKeyboard.PAGE_DOWN, 1)
    }

    override fun toString(): String {
        return "[LoadMoreArticle]"
    }
}
